monitoring/monitoring.go

```diff
@@ -52,35 +52,30 @@ const (
 	YearPeriod
 )
 
-const monitoringChannelSize = 64
+const (
+	monitoringChannelSize          = 64
+	capacityAlertProcessorElements = 5
+)
 
-/*******************************************************************************
+/***********************************************************************************************************************
  * Types
- ******************************************************************************/
+ **********************************************************************************************************************/
 
 // MonitoringAndAlertSender interface to send resource alerts.
-type MonitoringAndAlertSender interface { //nolint
-	SendResourceAlert(source, resource string, time time.Time, value uint64)
+type MonitoringAndAlertSender interface {
```

Comment:
Now it is used only for sending alert: rename to `AlertSender`.

Reply:
Done